<launch>

 <!--- Run AMCL --> 
    <include file="$(find amcl)/examples/amcl_diff.launch" />

   <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find multi_robot)/config/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
    <rosparam file="$(find multi_robot)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find multi_robot)/config/local_costmap_params.yaml" command="load" ns="pure_local_costmap" />
    <rosparam file="$(find multi_robot)/config/global_costmap_params.yaml" command="load" ns="pure_global_costmap" /> 
    <rosparam file="$(find multi_robot)/config/base_local_planner_params.yaml" command="load" ns="planner"/>
    <!-- <remap from="map" to="/map"/>
    <remap from="robot1/base_footprint" to="aofulbvasoubv" />
    <remap from="scan" to="robot1/scan" /> -->
 </node>

</launch> 
